Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -10,7 +10,7 @@ import Deferred
 
 private let MAP_SPAN_DELTA = 0.05
 private let MAP_LATITUDE_OFFSET = 0.015
-
+private let LOADING_TIMEOUT_IN_SECONDS: TimeInterval = 10
```

Comment:
>   I want our service query limitations to be reset back to zero so we can query our API's again.

Is this simply restarting our request to PlacesProvider or is there something else that I'm missing?

Hmm, I did  assume that we would show this every time we started loading places, but maybe that is a question for @vazquez